plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -165,6 +173,12 @@ func initQueries(s *SQLServer) error {
 		queries["AzureSQLDBPerformanceCounters"] = Query{ScriptName: "AzureSQLDBPerformanceCounters", Script: sqlAzureDBPerformanceCounters, ResultByRow: false}
 		queries["AzureSQLDBRequests"] = Query{ScriptName: "AzureSQLDBRequests", Script: sqlAzureDBRequests, ResultByRow: false}
 		queries["AzureSQLDBSchedulers"] = Query{ScriptName: "AzureSQLDBSchedulers", Script: sqlAzureDBSchedulers, ResultByRow: false}
+		// These two Query Store queries are not run by default as they are expensive and collection interval should be >=15m.
+		// If user needs data from Query Store, they will need to toggle the setting to true. By default it is turned off
+		if s.QueryStoreCollection == true {
```

Comment:
```suggestion
		if s.QueryStoreCollection {
```

Reply:
shouldn't this be based on the include/exclude queries instead?

Reply:
@ssoroka, this question is covered in the comment about config flag **query_store_collection**:
>is this something that can be covered by the exclude queries without adding a new boolean flag?

[https://github.com/influxdata/telegraf/pull/9150#discussion_r652166136](https://github.com/influxdata/telegraf/pull/9150#discussion_r652166136)